plugins/lsp/default.nix

```
@@ -242,15 +248,14 @@ lib.nixvim.plugins.mkNeovimPlugin {
           }"}
 
           for i, server in ipairs(__lspServers) do
-            local options = ${runWrappers cfg.setupWrappers "server.extraOptions"}
 
-            if options == nil then
-              options = __setup
-            else
-              options = vim.tbl_extend("keep", options, __setup)
-            end
+            vim.lsp.enable(server.name)
 
-            require("lspconfig")[server.name].setup(options)
+            vim.lsp.config(server.name, __setup)
+
+            if options then
+              vim.lsp.config(server.name, server.extraOptions)
+            end
```

This looks wrong. You're checking `options` but using `server.extraOptions`.

I think we want:

```suggestion
            if server.extraOptions then
              vim.lsp.config(server.name, server.extraOptions)
            end
```

?